### PR TITLE
[skip ci] mod_http_notification_doc

### PR DESCRIPTION
**Pull request by bartekgorny**
This PR addresses #728

### CODE REVIEW

File: doc/modules/mod_http_notification.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Module description
+
+This module enables forwarding certain events (messages, presence etc) to an
+external service via http.
+
+# How it works
+
+The module registers a `user_send_packe` hook - every time a user sends anything
```
Comment (michalwski) [inline doc/modules/mod_http_notification.md]: missing `t` in `user_send_packe`

**Comment by michalwski** [comment]
Did you try `mkdocs serve` to verify if the file is rendered correctly?

**Comment by bartekgorny** [comment]
No, I saw it rendered in my IntelliJ IDE and it looked fine.

**Comment by michalwski** [comment]
`mkdocs serve` shows you also if the file is visible in the right hand

**Pull request merged**